ticdc/troubleshoot-ticdc.md

```diff
@@ -270,3 +274,7 @@ If the Old Value feature is not enabled, you cannot tell whether a Row Changed E
 * `DELETE` event only contains the `"d"` field
 
 For more information, refer to [Open protocol Row Changed Event format](/ticdc/ticdc-open-protocol.md#row-changed-event).
+
+## How much PD storage does TiCDC use?
+
+TiCDC uses etcd in PD to store and regularly update the metadata. As the interval time between the MVCC of etcd and PD‘s default compaction is one hour, the amount of PD storage that TiCDC uses is proportional to the amount of metadata versions generated within this hour. However, in v4.0.5, v4.0.6, and v4.0.7, TiCDC has a problem of frequently writing, so if there are 1000 tables created or scheduled in an hour, it then takes up all the etcd storage and returns error `etcdserver: mvcc: database space exceeded`. You need to clean up the etcd storage after getting this error. See [etcd maintaince space-quota](https://etcd.io/docs/v3.4.0/op-guide/maintenance/#space-quota) for details. It is recommended to upgrade to v4.0.9 and later versions.
```

Comment:
```suggestion
TiCDC uses etcd in PD to store and regularly update the metadata. Because the time interval between the MVCC of etcd and PD‘s default compaction is one hour, the amount of PD storage that TiCDC uses is proportional to the amount of metadata versions generated within this hour. However, in v4.0.5, v4.0.6, and v4.0.7, TiCDC has a problem of frequent writing, so if there are 1000 tables created or scheduled in an hour, it then takes up all the etcd storage and returns the `etcdserver: mvcc: database space exceeded` error. You need to clean up the etcd storage after getting this error. See [etcd maintaince space-quota](https://etcd.io/docs/v3.4.0/op-guide/maintenance/#space-quota) for details. It is recommended to upgrade your cluster to v4.0.9 or later versions.
```